modules/beanmapping-dozer/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.devonfw.java.dev</groupId>
+    <artifactId>devon4j-modules</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>com.devonfw.java.modules</groupId>
+  <artifactId>devon4j-beanmapping-dozer</artifactId>
+  <version>${devon4j.version}</version>
+  <packaging>jar</packaging>
+  <name>${project.artifactId}</name>
+  <description>Minimal shim for bean mapping to convert between compatible Java beans (e.g. JPA entity to transfer-object and vice versa).</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.devonfw.java.modules</groupId>
+      <artifactId>devon4j-basic</artifactId>
+    </dependency>
+	 <dependency>
+	    <groupId>com.devonfw.java.modules</groupId>
+      <artifactId>devon4j-beanmapping</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.github.dozermapper</groupId>
+      <artifactId>dozer-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jaxb</groupId>
+      <artifactId>jaxb-runtime</artifactId>
+    </dependency>
```

Comment:
These dependencies came from the existing `devon4j-beanmapping` POM but do we actually know why these are needed at all for the beanmapping? Shouldnt dozer and orika come with all its dependencies to be functional? I think these dependencies are not required. Can we remove them? Or can we find out, why we need them and add some documentation as XML comment? If we need them, should they go to the api (`devon4j-beanmapping`) or to the impl (`devon4j-beanmapping-dozer` and `devon4j-beanmapping-orika`).

Reply:
tried to completely remove these dependencies from all the modules but it threw exception while running mvn install
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.112 s <<< FAILURE! - in com.devonfw.module.beanmapping.common.impl.dozer.BeanMapperImplDozerTest
[ERROR] testMapEntity2Eto  Time elapsed: 2.08 s  <<< ERROR!
com.github.dozermapper.core.MappingException: 
org.xml.sax.SAXException: javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]


Have kept the above dependency in devon4j-beanmapping and removed it from other modules devon4j-beanmapping-dozer and devon4j-beanmapping-orika).

Reply:
@vapadwal thanks for testing. Do you have the stacktrace at hand so we know the reason and can add it as XML comment?
Can you resolve the other review comment so we can complete and merge this?

Reply:
I assume that this is a bug of dozer only that does not declare these XML dependencies properly. This will maybe work in older Java versions (<=1.8) as they ship with JAXB included but since Java9 it has been removed from JDK. If that is the case, we would only need the dependency for dozer and orika does not need it.

Reply:
Thats correct only dozer has this problem, orika works